app/services/ImagePickerService.js

```diff
@@ -0,0 +1,23 @@
+import ImagePicker from 'react-native-image-picker';
+
+export const selectPhotoTapped = () => (
```

Comment:
how it should be used?
i.e. his approach is bad because i won't get visual response from a button:
```
<ImagePicker onPress={(localImage) => func(LocalImage)}>
  <Button title="Select Image"/>
</ImagePicker>
```